digits/extensions/data/imageGradients/forms.py

```diff
@@ -55,3 +56,37 @@ class DatasetForm(Form):
         default=32,
         validators=[validators.DataRequired()]
         )
+
+
+@subclass
+class InferenceForm(Form):
+    """
+    A form used to perform inference on a gradient regression model
+    """
+
+    gradient_x = utils.forms.FloatField(
+        'Gradient (x)',
+        validators=[
+            validate_required_iff(test_image_count=None),
+            validators.NumberRange(min=-1, max=1),
+            ],
+        tooltip="Specify a number between -1 and 1"
```

Comment:
Range should be [-0.5,0.5] instead. Otherwise you get this for (1,1):

![gradients-clipping](https://cloud.githubusercontent.com/assets/687269/18454183/caf17aca-78f7-11e6-8f48-9390012c4c16.jpg)